crates/router/src/db/refund.rs

```diff
@@ -638,61 +638,53 @@ mod storage {
             &self,
             merchant_id: &str,
             refund_details: &api_models::refunds::RefundListRequest,
-            storage_scheme: enums::MerchantStorageScheme,
+            _storage_scheme: enums::MerchantStorageScheme,
             limit: i64,
             offset: i64,
         ) -> CustomResult<Vec<diesel_models::refund::Refund>, errors::StorageError> {
-            match storage_scheme {
```

Comment:
Can we write a utility function which says what to use `Postgres` or `KV`, and in that function, we can decide based on feature. And use that function everywhere. So that we will not miss any OLTP